paasta_tools/kubernetes_tools.py

```diff
@@ -621,44 +616,6 @@ def get_autoscaling_metric_spec(
                         ),
                     )
                 )
-
-            use_signalfx = autoscaling_params.get(
```

Comment:
should we also clean up the use_prometheus (for uwsgi) bits above this?

Reply:
if we want, we can probably also clean up the metric names that we use for the HPAs since we no longer need to suffix things with `-prom` for uniqueness - we'd just need to update the prometheus adapter config script

that said, we should also update the prometheus adapter config script if we decide to remove use_prometheus from uwsgi codepaths

Reply:
yeah, getting rid of `use_prometheus` support is probably a good call.